osh/hub/settings_local.py

```diff
@@ -101,7 +101,7 @@
 # Root directory for uploaded files
 UPLOAD_DIR = os.path.join(FILES_PATH, 'upload')
 
-BZ_URL = "https://bugzilla.stage.redhat.com/xmlrpc.cgi"
+BZ_URL = "https://bugzilla.stage.redhat.com"
```

Comment:
do you think it's necessary to also update stage/prod settings in `osh-rh-conf` as well for consistency?

since `get_client` uses urljoin, leaving it unchanged won't do any harm.

Reply:
Other than that, everything looks good.

Reply:
Yes, the production configuration must be updated as well.

Reply:
I've prepared a MR with related modifications to `osh-rh-conf`.